app/build.gradle

```diff
@@ -25,6 +25,8 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation project(':data')
```

Comment:
Remove this dependency to data, app (presentation) doesn't need to know about it.

Reply:
Its true, is the domine the place where it  should be.